libs/partners/ai21/docs/provider.ipynb

```diff
@@ -0,0 +1,50 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# AI21\n",
+    "\n",
+    "AI21 is a platform that offers..."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {
+    "id": "y8ku6X96sebl"
+   },
+   "outputs": [],
+   "source": [
+    "from langchain_ai21.chat_models import AI21Chat\n",
+    "from langchain_ai21.llms import AI21LLM\n",
+    "from langchain_ai21.vectorstores import AI21VectorStore"
```

Comment:
This can't be right. Maybe remove the ipnbs that won't work

Reply:
Done